docs/source/java/substrait.rst

```diff
@@ -0,0 +1,149 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+=========
+Substrait
+=========
+
+Java Substrait offer capabilities to ``Query`` data received as a Susbtrait
+Plan (`plain or binary format`).
+
+During this process, Substrait plans are read, executed, and ArrowReaders are
+returned for reading Schema and ArrowRecordBatches. For Substrait plan that contains
+``Local Files`` the URI per table are defined in the Substrait plan, different
+than ``Named Tables`` where is needed to define a mapping name of tables
+and theirs ArrowReader representation.
+
+.. contents::
+
+Getting Started
+===============
+
+Java Substrait API uses Acero C++ Substrait API capabilities thru JNI wrappers.
```

Comment:
This section is not useful. Replace it with installation instructions or remove it.